tests/RotateTest.php

```diff
@@ -27,46 +27,61 @@ public function it_can_rotate_logs()
     {
         Log::info('test');
 
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log');
+	    $logs = glob(app()->storagePath().'/logs/*.log');
+	    foreach($logs as $log){
+		    $this->assertFileExists($log);
+	    }
 
         $resultCode = Artisan::call('logs:rotate');
 
         $this->assertEquals($resultCode, 0);
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.1.gz');
+
+	    foreach($logs as $log){
+		    $this->assertFileExists($log.'.1.gz');
+	    }
     }
 
     /** @test **/
     public function it_can_rotate_logs_withoutcompress()
     {
         Log::info('test');
 
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log');
+	    $logs = glob(app()->storagePath().'/logs/*.log');
+	    foreach($logs as $log){
+		    $this->assertFileExists($log);
+	    }
 
         $this->app['config']->set('rotate.log_compress_files', false);
 
         $resultCode = Artisan::call('logs:rotate');
 
         $this->assertEquals($resultCode, 0);
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.1');
+	    foreach($logs as $log){
+		    $this->assertFileExists($log.'.1');
+	    }
+
     }
 
     /** @test **/
     public function it_can_rotate_logs_with_maxfiles()
     {
-        $this->app['config']->set('rotate.log_compress_files', true);
-
-        for ($n = 0; $n < 10; $n++) {
-            file_put_contents(app()->storagePath().'/logs/laravel.log', 'test');
-            Artisan::call('logs:rotate');
-        }
-
-        $filesOld = glob(app()->storagePath().'/logs/laravel.log.*.gz');
-
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.1.gz');
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.2.gz');
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.3.gz');
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.4.gz');
-        $this->assertFileExists(app()->storagePath().'/logs/laravel.log.5.gz');
-        $this->assertFalse(file_exists(app()->storagePath().'/logs/laravel.log.6.gz'));
+	    $this->app['config']->set('rotate.log_compress_files', true);
+
+	    $logs = glob(app()->storagePath().'/logs/*.log');
+
+	    for ($n = 0; $n < 10; $n++) {
+		    foreach($logs as $log){
+			    file_put_contents($log, 'test');
```

Comment:
Never enter in this loop, becouse the variable $logs don't have any items and we get a warning in the test.

```bash
1) Cesargb\File\Rotate\Test\RotateTest::it_can_rotate_logs_with_maxfiles
This test did not perform any assertions
```

One solutions can be change the line #70 for this:
```php
$logs = [
            app()->storagePath().'/logs/laravel.log',
            app()->storagePath().'/logs/worked.log',
        ];
```

Can you change this or use other solutions?